src/patternfly/components/Form/form.scss

```diff
@@ -67,6 +67,8 @@ $pf-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "lg",
   --pf-c-form__label--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-form__label--LineHeight: var(--pf-global--LineHeight--sm);
   --pf-c-form__label--m-disabled--Color: var(--pf-global--disabled-color--100);
+  --pf-c-form__label--Cursor--hover: pointer;
```

Comment:
With component vars, the state (`hover`) in the var will be after the element the state applies to, and before the property that changes for that state. It follows the format of "when a `__label` is `hover`ed, set the `Cursor`"

More on that here - https://www.patternfly.org/v4/developer-resources/global-css-variables#component-variables

```suggestion
  --pf-c-form__label--hover--Cursor: pointer;
```

Reply:
@mcoker! Thanks a lot! My bad, I followed my intuition instead read the documentation you linked above. Excuse me :innocent: